Commands/Branding/AddJavaScriptBlock.cs

```diff
@@ -10,16 +10,22 @@ namespace SharePointPnP.PowerShell.Commands.Branding
     [CmdletHelp("Adds a link to a JavaScript snippet/block to a web or site collection",
         DetailedDescription = "Specify a scope as 'Site' to add the custom action to all sites in a site collection.",
         Category = CmdletHelpCategory.Branding)]
+    [CmdletExample(Code = "PS:> Add-SPOJavaScriptBlock -Name myAction -script '<script>Alert(\"This is my Script block\");</script>' -Sequence 9999 -Scope Site",
+                Remarks = "Add a JavaScript code block  to all pages within the current site collection under the name myAction and at order 9999",
+                SortOrder = 1)]
+    [CmdletExample(Code = "PS:> Add-SPOJavaScriptBlock -Name myAction -script '<script>Alert(\"This is my Script block\");</script>'",
+                Remarks = "Add a JavaScript code block  to all pages within the current web under the name myAction",
+                SortOrder = 2)]
     public class AddJavaScriptBlock : SPOWebCmdlet
     {
         [Parameter(Mandatory = true, HelpMessage = "The name of the script block. Can be used to identify the script with other cmdlets or coded solutions")]
         [Alias("Key")]
         public string Name = string.Empty;
 
-        [Parameter(Mandatory = true, HelpMessage = "The javascript block to add")]
+        [Parameter(Mandatory = true, HelpMessage = "The javascript block to add to the specified scope or site")]
         public string Script = null;
 
-        [Parameter(Mandatory = false)]
+        [Parameter(Mandatory = false, HelpMessage = "A sequence number where to add the script block")]
```

Comment:
The sequence defines the order in which the script blocks are added to the page. Maybe better to say "A sequence number that defines the order on the page" ?